test/framework/kogitotrusty.go

```diff
@@ -15,12 +15,41 @@
 package framework
```

Comment:
I was not sure where to put this logic... Theoretically, it belongs to Kogito Explainability, but we get the data using the Kogito Trusty... Let me know your thoughts about this and we can discuss from here.
cc @r00ta @sutaakar

Reply:
For me it's correct to keep the DTO in `kogitotrusty.go`, since we are interacting with the api exposed by such service

Reply:
fine for me too